node/cn/tracers/api.go

```diff
@@ -159,6 +159,10 @@ func (context *chainContext) GetHeader(hash common.Hash, number uint64) *types.H
 	return header
 }
 
+func (context *chainContext) Config() *params.ChainConfig {
```

Comment:
If blockchain.ChainContext.Config() becomes unnecessary, please remove it as well.

Reply:
https://github.com/kaiachain/kaia/pull/121#discussion_r1830537169